redaxo/src/addons/phpmailer/pages/checkmail.php

```diff
@@ -10,12 +10,12 @@
 $addon = rex_addon::get('phpmailer');
 
 $content = $smtpinfo = $mailerDebug = '';
-$emptymail = true;
 $date = new DateTime();
 if ('' == $addon->getConfig('from') || '' == $addon->getConfig('test_address')) {
-    $emptymail = false;
-}
-if ($emptymail = true) {
```

Comment:
Der wesentliche Fehler war hier die Zuweisung `=` statt `!=`.
In dem Zuge habe ich es aber noch vereinfacht und die Variable ganz entfernt.

Reply:
Wundert mich dass psalm/phpstan den fehler nicht erkannt haben

Reply:
Ich bin dran in Psalm RedundantCondition zu aktivieren, und dabei hat mir psalm diese Zeile gemeldet.